htdocs/web_portal/views/user/view_user.php

```diff
@@ -260,7 +260,7 @@
                     <?php
                         if(!$params['portalIsReadOnly'] &&
                             is_array($decorator) && $decorator["revokeButton"] == 'disabled') {
-                                echo 'Remove or reassign API credentials from site before revoking this role.';
+                                echo 'Remove or reassign API credentials from site(s) before revoking this role. Sites: ' . $decorator["revokeMessage"];
```

Comment:
Can this long line be broken up into shorter lines?

Reply:
Clarification: you mean for the UI or 'just' in the code?

Reply:
'just' in the code.

Reply:
Done.